WISE-Community/WISE
# Removed not null constraint from fields in Events table

How to verify:
1. Import wise_db_init.sql into a new database.
2. Log in as admin/pass, go to authoring tool, and make sure that you can create a new project.

Before, creating a new project threw an error inserting a new event in the Events table because of the not null constraint on the projectId field.

Fixes #1859.
